src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -65,6 +65,20 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({ lifecycleData, viewFilt
   const updatedLifecycleData: ChartDataObject[][] = [];
   const years: { [key: string]: Date } = {};
 
+  // Helper function to map backend support type to display name
+  const mapSupportTypeToDisplayName = (supportType: string, dataType: string): string => {
```

Comment:
Could this function be placed in module that could be shared between the two pages? That way it's only one place to update these values, since they are essentially constants.

Reply:
I think it's not needed at all -  the `LifecycleChart` and `LifecycleChartSystems` are identical files, just with different name. It's because of some bug and this was the solution for it. There is a lot of duplicit code anyway in files.